Week1/homework/js-exercises/10-compareArray.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const footballTeam = ['Goalkeeper', 'Defense player', 'Midfielder', 'Attack player'];
+const professionalFootballTeam = ['Goalkeeper','Sweeper (libero)','Defense player', 'Midfielder','Centre forward', 'Attack player'];
+
+console.log('The length of the first array is ...' + footballTeam.length);
+console.log('The length of the second array is ...' + professionalFootballTeam.length);
+
+
+
+if(footballTeam.length === professionalFootballTeam.length){
+
+    console.log('They are the same size!');
+
+}else if(footballTeam.length >> professionalFootballTeam.length){
```

Comment:
Ah, you can also just do this:

`footballTeam.length < professionalFootballTeam.length`

Reply:
I know I can do that but I liked the options to be more :)

Reply:
`>>` is not the greater than operator it is the bitwise operator, see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators

Reply:
The mistakes I made are only using keywords **var* instead of  **let or const**
It happened because I used to use it automatically. But I'll work on the notes.
Thanks guys.